tests/test_browser.py

```diff
@@ -3894,6 +3894,18 @@ def test_pthread_utf8_funcs(self):
   def test_pthread_wake_all(self):
     self.btest(path_from_root('tests', 'pthread', 'test_futex_wake_all.cpp'), expected='0', args=['-O3', '-s', 'USE_PTHREADS=1', '-s', 'TOTAL_MEMORY=64MB', '-s', 'NO_EXIT_RUNTIME=1'], also_asmjs=True)
 
+  # Test that real `thread_local` works.
+  @no_fastcomp('thread_local is only supported on WASM backend')
+  @requires_threads
+  def test_pthread_tls(self):
+    self.btest(path_from_root('tests', 'pthread', 'test_pthread_tls.cpp'), expected='1337', args=['-ftls-model=local-exec', '-s', 'PROXY_TO_PTHREAD', '-s', 'USE_PTHREADS', '-std=c++11'])
```

Comment:
Should `-ftls-model=local-exec` not be the default given that its the only one we support in llvm right now?   At least in emscripten would could pass it by default when you use `-s USE_PTHREADS`?

Reply:
After https://reviews.llvm.org/D64776, all thread-locals will be compiled as `local-exec` on Emscripten since we have no dynamic linking support with threads. I'll remove the explicit `-ftls-model=local-exec`.